pep8ext_naming.py

```diff
@@ -46,15 +46,18 @@ def __init__(self, class_name, bases, namespace):
             self._checks = []
 
 
-def _err(self, node, code):
+def _err(self, node, code, name=None):
     lineno, col_offset = node.lineno, node.col_offset
     if isinstance(node, ast.ClassDef):
         lineno += len(node.decorator_list)
         col_offset += 6
     elif isinstance(node, ast.FunctionDef):
         lineno += len(node.decorator_list)
         col_offset += 4
-    return (lineno, col_offset, '%s %s' % (code, getattr(self, code)), self)
+    code_str = getattr(self, code)
+    if name!=None:
```

Comment:
This should be written as

```py
if name is not None:
```

Per PEP-0008

Reply:
fixed